site/app/controllers/student/SubmissionController.php

```diff
@@ -65,11 +65,7 @@ private function showHomeworkPage() {
                          'action' => 'display',
                          'gradeable_id' => $gradeable->getId());
             $this->core->getOutput()->addBreadcrumb("<a href='{$this->core->buildUrl($loc)}'>{$gradeable->getName()}</a>");
-            
-            $select = $this->core->getOutput()->renderTemplate(array('submission', 'Homework'), 'gradeableSelect',
-                                                               $gradeable_list,
-                                                               $gradeable->getId());
-    
+                
```

Comment:
Remove this then from https://github.com/Submitty/Submitty/blob/030f0bdc23e9e2c97246ae57e24d72d4016ff1e5/site/app/controllers/student/SubmissionController.php#L71 and https://github.com/Submitty/Submitty/blob/master/site/app/views/submission/HomeworkView.php#L65 and https://github.com/Submitty/Submitty/blob/master/site/app/views/submission/HomeworkView.php#L70 and delete the function at https://github.com/Submitty/Submitty/blob/master/site/app/views/submission/HomeworkView.php#L33